inference-engine/cmake/features_ie.cmake

```diff
@@ -91,6 +91,8 @@ ie_option (ENABLE_OPENCV "enables OpenCV" ON)
 
 ie_option (ENABLE_PYTHON "enables ie python bridge build" OFF)
 
+ie_option (ENABLE_V7_SERIALIZE "enables serialization to IR v7" OFF)
```

Comment:
Why v7? If I'm not mistaken, `CNNNetwork::serialize` always produced IRv6

Reply:
It does not matter, it's "code name"